server/main.lua

```diff
@@ -103,7 +104,7 @@ lib.callback.register('qbx_management:server:hireEmployee', function(source, emp
         local organizationLabel = player.PlayerData[groupType].label
 		exports.qbx_core:Notify(source, Lang:t('success.hired_into', {who = targetFullName, where = organizationLabel}), 'success')
         exports.qbx_core:Notify(target.PlayerData.source, Lang:t('success.hired_to')..organizationLabel, 'success')
-        TriggerEvent('qb-log:server:CreateLog', logArea..'menu', grade, 'yellow', playerFullName..Lang:t('logs.recruited')..targetFullName..' ('..organizationLabel..')', false)
+		logger.log({source = source, event = 'hireEmployee', message = string.format('%s | %s hired %s into %s at grade %s', logArea, playerFullName, targetFullName, organizationLabel, grade)})
```

Comment:
Should the source be 'qbx_management'?

Also feel we should add a web hook to config that is nil but passed through for those who want discord logging.